python/gtsam/tests/test_HybridFactorGraph.py

```diff
@@ -11,75 +11,167 @@
-                                   noiseModel)
-        jf2 = gtsam.JacobianFactor(X(0), np.eye(3), np.ones((3, 1)),
-                                   noiseModel)
+        jf1 = JacobianFactor(X(0), np.eye(3), np.zeros((3, 1)), model)
+        jf2 = JacobianFactor(X(0), np.eye(3), np.ones((3, 1)), model)
 
-        gmf = gtsam.GaussianMixtureFactor.FromFactors([X(0)], dk, [jf1, jf2])
+        gmf = GaussianMixtureFactor([X(0)], dk, [jf1, jf2])
 
-        hfg = gtsam.HybridGaussianFactorGraph()
-        hfg.add(jf1)
-        hfg.add(jf2)
+        hfg = HybridGaussianFactorGraph()
+        hfg.push_back(jf1)
+        hfg.push_back(jf2)
         hfg.push_back(gmf)
 
         hbn = hfg.eliminateSequential(
-            gtsam.Ordering.ColamdConstrainedLastHybridGaussianFactorGraph(
-                hfg, [C(0)]))
+            Ordering.ColamdConstrainedLastHybridGaussianFactorGraph(hfg, [C(0)])
+        )
 
         self.assertEqual(hbn.size(), 2)
 
         mixture = hbn.at(0).inner()
-        self.assertIsInstance(mixture, gtsam.GaussianMixture)
+        self.assertIsInstance(mixture, GaussianMixture)
         self.assertEqual(len(mixture.keys()), 2)
 
         discrete_conditional = hbn.at(hbn.size() - 1).inner()
-        self.assertIsInstance(discrete_conditional, gtsam.DiscreteConditional)
+        self.assertIsInstance(discrete_conditional, DiscreteConditional)
 
     def test_optimize(self):
         """Test construction of hybrid factor graph."""
-        noiseModel = gtsam.noiseModel.Unit.Create(3)
-        dk = gtsam.DiscreteKeys()
+        model = noiseModel.Unit.Create(3)
+        dk = DiscreteKeys()
         dk.push_back((C(0), 2))
 
-        jf1 = gtsam.JacobianFactor(X(0), np.eye(3), np.zeros((3, 1)),
-                                   noiseModel)
-        jf2 = gtsam.JacobianFactor(X(0), np.eye(3), np.ones((3, 1)),
-                                   noiseModel)
+        jf1 = JacobianFactor(X(0), np.eye(3), np.zeros((3, 1)), model)
+        jf2 = JacobianFactor(X(0), np.eye(3), np.ones((3, 1)), model)
 
-        gmf = gtsam.GaussianMixtureFactor.FromFactors([X(0)], dk, [jf1, jf2])
+        gmf = GaussianMixtureFactor([X(0)], dk, [jf1, jf2])
 
-        hfg = gtsam.HybridGaussianFactorGraph()
-        hfg.add(jf1)
-        hfg.add(jf2)
+        hfg = HybridGaussianFactorGraph()
+        hfg.push_back(jf1)
+        hfg.push_back(jf2)
         hfg.push_back(gmf)
 
         dtf = gtsam.DecisionTreeFactor([(C(0), 2)], "0 1")
-        hfg.add(dtf)
+        hfg.push_back(dtf)
 
         hbn = hfg.eliminateSequential(
-            gtsam.Ordering.ColamdConstrainedLastHybridGaussianFactorGraph(
-                hfg, [C(0)]))
+            Ordering.ColamdConstrainedLastHybridGaussianFactorGraph(hfg, [C(0)])
+        )
 
         hv = hbn.optimize()
         self.assertEqual(hv.atDiscrete(C(0)), 1)
 
+    @staticmethod
+    def tiny(num_measurements: int = 1):
+        """Create a tiny two variable hybrid model."""
```

Comment:
I would like to see this say "Create a tiny two variable hybrid model which represents the generative probability P(z, x, n) = P(z | x, n)P(x)P(n)."